ob_v3p0/impl/conformance.md

```diff
@@ -0,0 +1,42 @@
+## Conformance and Certification
+
+Open Badges and CLR enable implementers to choose specific roles their product
+fills and receive certification for the capabilities of that role.
+
+### Certified Roles in Open Badges
+
+TODO the Conformance and Certification Guide in this repo lists the roles as
+"Issuer", "Displayer", "Host" in one spot, and then Service Consumer (Read),
+Service Consumer (Write), Service Provider (Read), Service Provider (Write), and
+it hasn't yet incorporated the idea of "issuer only" certification that was
+referenced in recent meetings. Follow up and make consistent.
```

Comment:
may be captured in the todo, but would be helpful to incorporate VC terminology such as Issuer and Holder into the below bullets.